src/Operation/Aggregate.php

```diff
@@ -251,7 +262,7 @@ public function execute(Server $server)
             ? $server->executeReadWriteCommand($this->databaseName, $command, $options)
             : $server->executeReadCommand($this->databaseName, $command, $options);
 
-        if ($this->options['useCursor']) {
+        if ($this->options['useCursor'] || ( ! $this->options['useCursor'] && ! empty($this->options['explain']))) {
```

Comment:
`! $this->options['useCursor']` can be factored out here, since we'd only reach the right-hand side of `||` if `useCursor` was not true.